### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by r-brown**
Add netlicensing.io / BaaS / LaaS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -511,7 +511,7 @@ Table of Contents
   * [paraio.com](https://paraio.com) — Backend service API with flexible authentication, full-text search and caching. Free for 1 app, 1GB app data.
   * [remotemysql.com](https://remotemysql.com) — Remote MySQL Database hosting, setup is instant and use phpMyAdmin for administration, free for 100Mb data, free backups, no query limits and 99% uptime.
   * [ably.com](https://www.ably.com) - APIs for realtime messaging, push notifications, and event-driven API creation. Free plan has 3m messages/mo, 100 concurrent connections, 100 concurrent channels.
-
+  * [netlicensing.io](https://netlicensing.io) - A cost-effective and integrated Licensing-as-a-Service (LaaS) solution for your software on any platform from Desktop to IoT and SaaS. Basic Plan for *FREE* while you are a student.
```
Comment (ripienaar) [inline README.md]: Doesnt seem to be just free for students? There's a basic free tier for everyone?

Comment (r-brown) [reply README.md]: > Doesnt seem to be just free for students? There's a basic free tier for everyone?

Yes, it is - NetLicensing offers all "GitHub Student Developer Pack" participants a free _Basic Plan_ - https://netlicensing.io/github-students/
Which is more than _FREE forever_

Comment (ripienaar) [reply README.md]: sure but what i am saying is they also have a free tier thats not for students, as its written now it seems almost like its just for students when in fact the free tier also applies to others. So we could clarify and more people might look at it

Comment (r-brown) [reply README.md]: It's written like that:

> Basic Plan for *FREE* while you are a student.

Also:
- FREE forever - for everyone
- Basic Plan - for students
  - more options available

=> also, the students have a clear advantage here.

**Comment by ripienaar** [comment]
as you wish.

**Pull request merged**